File: hw/sanchez/u3/HW20TemplateMethod/TemplatePY/ec/edu/espe/cafe/BeverageTest.py
from model.Tea import Tea
from model.Coffee import Coffee

def main():
    tea = Tea()
    coffee = Coffee()

    print("\nMaking tea...")
    tea.prepareRecipe()

    print("\nMaking coffee...")
    coffee.prepareRecipe()

if __name__ == "__main__":
    main()

File: hw/andrade/u3/HW21TemplateMethod/HW21TemplateMethod/src/main/java/ec/edu/espe/hw21templatemethod/view/BeverageTest.java
package ec.edu.espe.hw21templatemethod.view;

import ec.edu.espe.hw21templatemethod.model.Coffee;
import ec.edu.espe.hw21templatemethod.model.Tea;

/**
 *
 * @author Micaela Jacome DESKTOP-46VMNHU ESPE
 */
public class BeverageTest {

    public static void main(String[] args) {
        Tea tea = new Tea();
        Coffee coffee = new Coffee();
        
        System.out.println("\nMaking tea...");
        tea.prepareRecipe();
        
        System.out.println("\nMaking coffee...");
        coffee.prepareRecipe();
    }
}
